examples/nuxt/.npmrc

```diff
@@ -1 +0,0 @@
-shamefully-hoist=true
```

Comment:
I think we still need this for pnpm: https://github.com/nuxt/nuxt/issues/14146.

(I mention that in case users clone this directory.)